Foosball/Foosball/BallTracker.cs

```diff
@@ -14,32 +14,31 @@ namespace Foosball
     public partial class BallTracker : Form
     {
         private Capture _capture;
+       
         private Deque<Point> _deque = new Deque<Point>();
         private int _hLow = 0;
         private int _sLow = 43;
         private int _vLow = 209;
         private int _hHigh = 13;
         private int _sHigh = 255;
         private int _vHigh = 255;
+        private MatchController _matchController;
+        Mat m = new Mat();
```

Comment:
This m could be changed into something more explainable (for example _mat), and it could also be private.